edifact/outgoing/models/message.py

```diff
@@ -166,7 +166,7 @@ def __init__(self, transaction_type: TransactionType):
 
 class ReferenceTransactionNumber(Reference):
 
-    def __init__(self, reference=''):
+    def __init__(self, reference=(None, int)):

```

Comment:
should be `reference=None: (None, int)` for the type parameter

this is setting the default value of parameter reference to the tuple `(None, int)`

Reply:
reference: (int, None) = None
DONE